api/src/main/javadoc/doc-files/EFSL.html

```diff
@@ -20,7 +20,7 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
   <li>All existing copyright notices, or if one does not exist, a notice
       (hypertext is preferred, but a textual representation is permitted)
       of the form: &quot;Copyright &copy; [$date-of-document]
-      &ldquo;Eclipse Foundation, Inc. &lt;&lt;url to this license&gt;&gt;
+      &ldquo;Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php
```

Comment:
```suggestion
      Eclipse Foundation, Inc. 
      <a href="https://www.eclipse.org/legal/efsl.php">https://www.eclipse.org/legal/efsl.php</a>
```
Anchor tag is missing hence it is not rendered as link. There is a leading quote before Eclipse Foundation.

Reply:
I don't think it needs to be rendered as an anchor. @kwsutter do you think that is necessary?

For now, let's stick with what is currently present in the PR (unless we hear otherwise) and if necessary let's add another change later.

Reply:
IMHO you should not modify the reference license text, but instead create your own notice file based on the template provided in there. 

@SharonCorbett would you agree? I am adding this to ScanCode-toolkit for license detection and it probably is OK to have variants of the notices, but to have variants of the license text proper?

Reply:
@pombredanne Let me investigate and get back to you.  There should be no variant to the license text but to ensure we get the notice right before you add to ScanCode-toolkit, I'd like time to investigate further!  I'll be back here by close of business tomorrow!  TIA!

Reply:
@pombredanne:  Thanks for checking in on this.  As indicated there is to be no modification of the license text itself, it is to be used in its exact form as per https://www.eclipse.org/legal/efsl.php.  As you suggest, only the notice can have a variant as per the terms of the EFSL itself.  

For those on this pull request, please ensure you are adhering.  

If there are any lingering questions, please contact license@eclipse.org.

Regards,
Sharon